main/webapp/modules/core/scripts/reconciliation/standard-service-panel.js

```diff
@@ -98,6 +99,22 @@ ReconStandardServicePanel.prototype._constructUI = function() {
   this._elmts.typeInput.attr('aria-label',$.i18n('core-recon/type'))
 
   this._elmts.rawServiceLink.attr("href", this._service.url);
+// Hide the documentation link initially
+this._elmts.documentationLink.css("display", "none");
```

Comment:
I would suggest to refector this so that the documentation link is inserted into the DOM if there is a link.

It's more aligned with the rest of the code base and would get rid of issues like, leftover elements and non-default display values.